.src/build.ts

```diff
@@ -276,16 +247,26 @@ for (const dir of ALL_DIST_DIRS) {
 // -----------------------------------------------------------------------------
 {
   packageJson.version = VERSION;
-  packageJson.main = `./draft-${nodeLatestDraft}.js`;
+  packageJson.main = `./draft_${latestDraft}.js`;
+  packageJson.types = `./draft_${latestDraft}.d.ts`;
   // @ts-expect-error this is valid
   packageJson.exports = {
-    ".": `./draft-${nodeLatestDraft}.js`,
+    ".": {
+      types: `./draft_${latestDraft}.d.ts`,
+      default: `./draft_${latestDraft}.js`,
+    },
     ...Object.fromEntries(
       drafts.map((
         draftId,
       ) => {
         const nodeDraftId = draftId.replaceAll("_", "-");
-        return [`./draft-${nodeDraftId}`, `./draft-${nodeDraftId}.js`];
+        return [
+          `./draft-${nodeDraftId}`,
```

Comment:
I tried to keep the public API the same, which is why I didn't change the export names.